docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -195,6 +195,7 @@ Open the `.env` file you just created and update the following environment varia
 | Environment variable | Description | Reference location |
```

Comment:
Can you also add a section for deploying to Vercel? When doing that, you only need to set the DATABASE_URL to start, with the remaining variable set as placeholders (i.e. using '1234' for each)

Reply:
@becomevocal thank you for this information. I updated the article. Can you please review?